docs/getting-started.md

```diff
@@ -6,7 +6,29 @@ nav_order: 2
 
 # Getting Started
 
-**Authelia** can be tested in a matter of seconds with Docker and docker-compose.
+## Docker Compose
```

Comment:
IMO this getting started should contain what is in docs/deployment/deployment-local.md. docs/deployment/deployment-local.md should not exist. Only lite, ha and kube should exist in the deployments section.
And at the end of the getting started, I would add links to those documents in deployments section.

In my mind, the profile of people I picture who will use this documentation is someone knowing nothing about Authelia and potentially nothing about CS either. This person just wants to see how Authelia looks like not based on README pictures and how to use it for real. The basic use case that I can picture is the protection of a ownCloud or a personal website but the user does not necessarily know what is DNS, reverse proxies, ports. She just ran docker run owncloud that she read in a blog post or something.

So here, in that Getting Started, I'd rather take the user by the hand and show what Authelia looks like for real in the browser by taking the user to protected websites (one single-factor and one two-factor as you did is great) and eventually show how to customize one ACL rule. This will give the best overview that will, or not, ring a bell to the user. Consequently, she will, or not, proceed with lite or HA deployment. That's why I think those should be mentioned at the end and we should provide a link to their related deployment documentation.

Do you see what I mean?